roles/katello_devel/meta/main.yml

```diff
@@ -2,6 +2,7 @@
 dependencies:
   - role: etc_hosts
```

Comment:
I'd expect this to also need become true?

Reply:
I had the same thought, but `become: yes` is already specified at the task level within that role.